Python/Product/PythonTools/PythonTools/Options/ExperimentalOptions.cs

```diff
@@ -0,0 +1,54 @@
+// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using EO = Microsoft.PythonTools.Interpreter.ExperimentalOptions;
+
+namespace Microsoft.PythonTools.Options {
+    public sealed class ExperimentalOptions {
+        internal ExperimentalOptions(PythonToolsService service) {
```

Comment:
`service` isn't used, did you leave it for possible use by later options?

Reply:
Yeah, and consistency with the other options classes. We only have one experimental option right now and it is system-wide, but future ones could be per-instance, in which case we should use the normal options infrastructure to store the setting.